### PR TITLE
chore(rebuild): re-incorporate redirects

### PR DESCRIPTION
**Pull request by EugeneHlushko**
Re-intergrate redirects

Note: i took latest redirects list from `master`

### CODE REVIEW

**Comment by montogeek** [comment]
What happened to the redirects.json file?

File: DISCUSSION.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ this branch:
 - [ ] Extract anchors into `_content.json` via `DirectoryTreePlugin`
 - [ ] Finish re-incorporating mobile sidebar
 - [ ] Re-integrate google-analytics
-- [ ] Re-incorporate `redirects.json`
+- [x] Re-incorporate `redirects.json` (Eugene)
```
Comment (skipjack) [inline DISCUSSION.md]: Can you drop this change to fix the conflict? I removed `DISCUSSION.md` in favor of github threads (i.e. PRs and issues). It wasn't meant to be a long-lived document anyway, just gave some perspective on various issues.

**Comment by Munter** [comment]
When the hyperlink branch merges in you won't have any links pointing at moved pages any more. I'm also pretty sure we already addressed all of them on master, but if there are any left, rest assured you'll have automation to find them

**Comment by EugeneHlushko** [comment]
@skipjack its done

**Comment by skipjack** [comment]
@EugeneHlushko hope you don't mind, I tested this out before merging and noticed that plugin was actually run twice because of our parallelized `prod` configs. By moving it next to the `SSGPlugin`, we only run it once and lump together all HTML generation into a single build.

I'm going to merge, but if you see any issue with this change don't hesitate to comment.

**Pull request merged**